packages/demux/src/DataAdapter.ts

```diff
@@ -60,4 +60,18 @@ export interface DataAdapter<ID = any, D = any> {
    * @param meta metadata (to be updated)
    */
   delete(name: modelName, id: ID, meta: DMeta): Promise<boolean>;
+
+  /**
+   * revert data to state of blockNumber in case of fork
+   * @param blockNumber the point to revert to
+   * revert data will be from blockNumber + 1 to current processed block
+   */
+  revert?(blockNumber): Promise<any>;
```

Comment:
I don't think revert and deleteAuditHistory should exist in the DataAdapter layer.

Reply:
So could you recommend how to implement revert?